subtitle/subtitle_downloader.py

```diff
@@ -106,6 +107,17 @@ def download_subtitle(video):
         subtitle_list = SubtitleDownloader.fetch_subtitle_list(cid)
         if not subtitle_list:
             logging.info(u"No subtitle available on the server.")
+            logging.info(u"Falling back to search by filename".format(video))
+            subtitle = ""
+            try:
+                subtitle = subprocess.check_output(["opensubtitles_downloader", "-f", video]).decode("utf-8")
```

Comment:
I think running an external program is the last choice to do it. Maybe importing or implementing the feature inside the package is a better choice?

Reply:
I guess from an integration perspective it's the best choice, but I also think calling the program directly results in less Loc. Anyway, I'll look into it when I have time.